README.md

```diff
@@ -50,6 +50,23 @@ cache-control: no-cache
 
 Will post two messages across two paritions to the Event Hub specified by the `EventHubConnection` and `EventHubName` settings in your `local.settings.json` file or - when published to Azure - the Function App's application settings.
 
+## Event Hubs Kafka
+
+```http
+POST /api/PostToEventHubKafka HTTP/1.1
+Content-Type: application/json
+cache-control: no-cache
+
+{
+  "NumberOfMessagesPerPartition": 2
+}
+```
+
+Will post two messages per partition of the Event Hub specified by the `EventHubKafkaConnection`, `EventHubKafkaName` and `EventHubKafkaFQDN` settings in your `local.settings.json` file or - when published to Azure - the Function App's application settings.
+The number of messages per partition will differ by no more than 1.

```

Comment:
I'm not sure what this means - esp since the name of the property is "NumberOfMessagePerPartition" - can you elaborate in this doc?

Reply:
Added the following lines:
Which means that at any given point of time, if the number of messages expected to be sent per partition is say `x`, then you can expect that there may be some partitions with number of messages `x+1` and some with number of messages `x-1`.